crates/holochain_conductor_api/src/config/conductor/dpki_config.rs

```diff
@@ -1,13 +1,43 @@
 // Legacy config that will probably change
 #![allow(missing_docs)]
 
+use std::path::PathBuf;
+
 use serde::Deserialize;
 use serde::Serialize;
 
 /// Configure which app instance ID to treat as the DPKI application handler
 /// as well as what parameters to pass it on its initialization.
 #[derive(Clone, Deserialize, Serialize, Debug, PartialEq)]
 pub struct DpkiConfig {
-    pub instance_id: String,
-    pub init_params: String,
+    /// Path to a DNA which implements the DPKI service, i.e. Deepkey.
+    /// Defaults to the built-in Deepkey DNA from the holochain_deepkey_dna crate.
+    pub dna_path: Option<PathBuf>,
+
+    /// The lair tag used to refer to the "device seed" which was used to generate
+    /// the AgentPubKey for the DPKI cell
+    pub device_seed_lair_tag: String,
+
+    /// For testing only, we can turn off DPKI if needed.
+    /// TODO: this can be removed once DPKI is truly optional again.
+    #[serde(default)]
+    pub no_dpki: bool,
```

Comment:
```suggestion
    /// For testing only, we can turn off DPKI if needed.
    /// TODO: this can be removed once DPKI is truly optional again.
    #[serde(default)]
    #[cfg(test)]
    pub no_dpki: bool,
```

Reply:
I'd prefer this be publicly available for those who know what they're doing. For instance if something is borked with DPKI and we need to run a hackathon.